spec/translation_spec.rb

```diff
@@ -27,6 +27,29 @@ def to_s
   specify { expect(subject.to_s).to eq 'My name is Ruby' }
   specify { expect(class_translation).to eq 'My name is Class' }
 
+  specify {
+    allow(TranslationModel).to receive(:base_translation_class).and_call_original
```

Comment:
For consistency, these two lines setting up the context should be in a before block within a context